### PR TITLE
LPS-158124 Add new gradle task 'pruneDockerImage'

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:relevant

**Comment by Seiphon** [comment]
ci:test:gradle-plugins

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java [hunk 1]
```diff
@@ -460,9 +471,15 @@ public void execute(Project p) {
 
 					setProperty.add(dockerImageId);
 
-					Property<String> property = dockerRemoveImage.getImageId();
+					Property<String> removeImageProperty =
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java:474]: hey @Seiphon  can you sort this?  Get and Set the `pruneImageProperty` before the `removeImageProperty`
and lets call it
`Property<String> pruneImageIdProperty`
`Property<String> removeImageIdProperty`

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import com.github.dockerjava.api.command.PruneCmd;
+import com.github.dockerjava.api.model.PruneType;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.provider.Property;
+
+/**
+ * @author Seiphon Wang
+ */
+public class DockerPruneImage extends DockerExistingImage {
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java:29]: @Seiphon  looks good.  Can you add a test?

**Review by gamerson — CHANGES_REQUESTED** [review]

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java (hunk 2 above)
Comment (Seiphon) [reply modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java:29]: OK Greg, I will add it and resend the pr

**Pull request closed without merging**